public/packs/manifest.json

```diff
@@ -1,14 +1,24 @@
 {
-  "application.js": "/packs/js/application-9afcbb5693aa87623e69.js",
-  "application.js.map": "/packs/js/application-9afcbb5693aa87623e69.js.map",
+  "application.js": "/packs/js/application-f3151c7bc3b441a5f9bf.js",
```

Comment:
👀

Reply:
I can probably also omit this file - just wanted to be sure that it wouldnt cause anything to break